saphir/src/multipart/parser.rs

```diff
@@ -313,7 +314,7 @@ mod tests {
         let boundary_partial = &b"------AaB"[..];
         let empty = &b""[..];
 
-        assert_eq!(tag_boundary(boundary)(boundary_partial), Err(nom::Err::Incomplete(Needed::Size(10))));
+        assert_eq!(tag_boundary(boundary)(boundary_partial), Err(nom::Err::Incomplete(Needed::Size(NonZeroUsize::new(3).unwrap()))));
```

Comment:
The "breaking changes" section for the 6.0 release (https://github.com/Geal/nom/blob/master/CHANGELOG.md#breaking-changes) mentions that "streaming parsers return the number of additional bytes they need, not the total. This was supposed to be the case everywhere, but some parsers were forgotten". Looking at the test, it seems that the correct value should indeed have been 3 and not 10.